src/screens/Topics/index.js

```diff
@@ -316,21 +278,21 @@ const styles = StyleSheet.create({
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: '400',
-    fontSize: 10,
+    fontSize: normalizeFontSize(10),
     textAlign: 'center',
     color: colors.blackgrey
   },
   button: {
     backgroundColor: colors.gray
   },
   scrollButtonParent: {
-    paddingHorizontal: 22
+    paddingHorizontal: dimen.normalizeDimen(22)
   },
   containerContent: {
-    paddingRight: 20
+    paddingRight: dimen.normalizeDimen(20)
   },
   reloadTopicButtonContainer: {
-    marginTop: 60
+    marginTop: dimen.normalizeDimen(60)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk are all about replacing hard-coded values with normalized dimensions using the `dimen.normalizeDimen` function and adjusting various style and layout properties such as margins, padding, widths, heights, and font sizes. These changes will improve the responsiveness and visual consistency of the components across different screen sizes and resolutions.
```diff
+     fontSize: normalizeFontSize(10),
...
+     paddingHorizontal: dimen.normalizeDimen(22)
...
+     paddingRight: dimen.normalizeDimen(20)
...
+     marginTop: dimen.normalizeDimen(60)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->